### PR TITLE
First pass at fleshing out the glossary

### PR DESCRIPTION
**Pull request by slack**
This is based off the work that @arschles started in https://github.com/kubernetes-incubator/service-catalog/pull/15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +18,36 @@ those Service Brokers.
 the Service Catalog requesting a new Service Instance from the owning Service
 Broker.
 
+**Application**: code that will access or consume a Service. While in Kubernetes
+the code that is deployed is often called a "Service", to avoid confusion, this
+document will refer to the code that accesses a service as an "application".
+
+**Application operator**: the person or team responsible for deploying a
+consumer application. Users in this role, at minimum, have access to their own
```
Comment (duglin) [inline README.md]: s/consumer//

File: docs/v1/use-cases.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ TODO: add glossary
 TODO: scrub doc post glossary
 TODO: add CF SB API as the lingua franca for blackbox services
 
+## Personas
+
+* User
+* Developer
```
Comment (duglin) [inline docs/v1/use-cases.md]: I suspect that just "Application Manager" (or Operator perhaps) might be sufficient to cover both of these.

File: docs/v1/use-cases.md [hunk 3]
```diff
@@ -4,6 +4,13 @@ TODO: add glossary
...
+* Broker Operator
+* Service Provider
```
Comment (duglin) [inline docs/v1/use-cases.md]: For blackbox I would suggest that we don't draw a distinction between the person who manages the service broker and the person who owns the backend service - all we see is the service broker. Whether we call that  "Broker Operator" or "Service Provider"....either works but Service Provider is the term I'm more used to.

**Comment by duglin** [comment]
ah geez - you erase the 2nd file right as I was commenting on it :-)

**Comment by slack** [comment]
Hah, sorry. I accidentally committed it to the wrong working branch. Too much juggling.

**Comment by duglin** [comment]
squash but LGTM

**Comment by arschles** [comment]
@duglin are you generally ok with using the github squash and merge functionality?

**Comment by pmorie** [comment]
LGTM for now, merging.

**Comment by duglin** [comment]
@arschles just a huge personal preference but I prefer one commit per PR, unless there's a really good reason that people need to see the work split up - but to be honest, I think those times are pretty rare.  So, yes, IMO all PRs should be squashed.  I'm less interested in seeing how the sausage is made, I just want to see the net result.

**Comment by pmorie** [comment]
I am also a militant squasher.  Keep the git history clean!

**Comment by duglin** [comment]
militant - I like that term.

**Comment by arschles** [comment]
Ok, fair enough on the "see how the sausage is made" point.

But the squash-on-merge functionality keeps the commit history in master clean fwiw.

**Comment by duglin** [comment]
interesting, I didn't know github has a squash-on-merge option - something new for a Friday :-)

**Pull request merged**